miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -283,6 +284,11 @@ public static SampleAdditionalInfo to(SampleAdditionalInfoDto from) {
       sampleClass.setSampleClassId(from.getSampleClassId());
       to.setSampleClass(sampleClass);
     }
+    if (from.getParentId() != null) {
```

Comment:
Should we get rid of this line then? (It's certainly possible that I was overgeneralizing here)